### PR TITLE
first pass at kepler release notes draft

### PR DESCRIPTION
**Pull request by wteouchicago**
- Release notes based on current portal RC of Kepler for UC.

Figured it's good to have a draft in so we can easily incorporate SJ's notes. 

I left SJ's 508 known issues as-is from 2.0 since I don't think there are many changes in that are for 2.1, but I removed their other known issues since they've been making a lot of changes there.

### CODE REVIEW

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 | Version | Date |
 |---|---|
+| [v2.1.0](Data_Portal_Release_Notes.md#release-210) | April 30, 2024 |
```
Comment (wteouchicago) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:5]: we'll hv to change this if launch date changes.
I assumed we would call it 2.1.0, but up to Bill, really.

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 2]
```diff
@@ -36,6 +37,88 @@
 | [v1.0.1](Data_Portal_Release_Notes.md#release-101) | May 18, 2016 |
 
 ---
+## Release 2.1.0
+
+* __GDC Product__: GDC Data Portal
+* __Release Date__:  April 30, 2024
```
Comment (wteouchicago) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:43]: note the date will have to change if launch date changes

**Review by ebusygina — APPROVED** [review]

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md (hunk 1 above)
Comment (wwysoc2) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:5]: 2.1 is good

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 3]
```diff
@@ -36,6 +37,88 @@
...
+* Files in the Repository can now be filtered by Tissue Type, Tumor Descriptor, Specimen Type, and Preservation method. <!--PEAR-1514-->
+* Metadata and Sample Sheet downloads have been added to the Repository. <!--PEAR-1863-->
+* The performance of the Clinical Data Analysis tool has been improved, especially when large cohorts are used with QQ plots. <!--PEAR-1536-->
+* Quick Search now returns results for the latest versions of files when searching for older versions of those files. <!--PEAR-1804--> 
+* Responsiveness improvements have been made to the Analysis Center and the Cohort Bar. <!--PEAR-1836-->
+* The X button on the Unexpected Error dialog box has been removed. <!--SV-2367-->
+* Buttons for launching demos have been removed from the selection view of Cohort Comparison and Set Operations. <!--SV-2328/2327-->
+* The UX/UI for the Cohort Builder has been improved. <!--PEAR-1547-->
+* Set names for sets of the same type are now enforced to be unique when editing names in Manage Sets. <!--PEAR-1359-->
+* Number ranges cards in the Cohort Builder no longer display the custom range option when there is no data. <!--PEAR-661-->
```
Comment (wwysoc2) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:57]: "Number range cards"  (not sure if it needs to be singular).

**Review by wwysoc2 — APPROVED** [review]
LGTM with some minor changes

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 4]
```diff
@@ -36,6 +37,88 @@
...
 
 ---
+## Release 2.1.0
+
+* __GDC Product__: GDC Data Portal
+* __Release Date__:  April 30, 2024
+
+### New Features and Changes
+* The Annotations Browser and annotation summary page have been implemented. <!--PEAR-1838/1181/1182/1160--> 
+* The case summary page has been enhanced with a table listing all the files associated with the case. Additionally, a link to the table is now available in the header of the summary page, and information has been added to the File Counts summary tables  to lead users to the new files table. The clinical and biospecimen supplements tables have also been removed from the case summary page. <!--PEAR-1822/1849/1833/PEAR-1832-->
```
[the remark this answers is not in the document]
Comment (wteouchicago) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md]: good catch! fixed.

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 5]
```diff
@@ -36,6 +37,88 @@
...
+* Files in the Repository can now be filtered by Tissue Type, Tumor Descriptor, Specimen Type, and Preservation method. <!--PEAR-1514-->
```
[the remark this answers is not in the document]
Comment (wteouchicago) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md]: done!

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md (hunk 3 above)
Comment (wteouchicago) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md]: fixed! not sure if it needs to be singular either, but does sound better.

**Pull request merged**